### PR TITLE
Complete openapi documentation

### PR DESCRIPTION
**Pull request by attwad**
I think this is final this time :) Can't think of anywhere to add new descriptions or examples...

Also make it look nicer by using themes.
https://github.com/ostranme/swagger-ui-themes#monokai

### CODE REVIEW

File: verification/curator-service/api/openapi/openapi.yaml [hunk 1]
```diff
@@ -955,16 +983,38 @@ components:
                 $ref: "#/components/schemas/DateRange"
               value:
                 type: string
+                description: >
+                  For outcome event, value can be Death or Recovered.
+                  For icuAdmission and hospitalAdmission, value can be Yes if the patient was hospitalized.
```
Comment (allysonjp715) [inline verification/curator-service/api/openapi/openapi.yaml:988]: Maybe 
`For icuAdmission and hospitalAdmission, values can be Yes or No.`
to indicate they can specify if they know the case wasn't hospitalized

File: verification/curator-service/api/openapi/openapi.yaml [hunk 2]
```diff
@@ -983,14 +1039,23 @@ components:
...
+              example:
+                - Oral
+                - Air-borne
             places:
               type: array
               items:
                 type: string
+              example:
+                - Airplane
+                - Bub
```
Comment (allysonjp715) [inline verification/curator-service/api/openapi/openapi.yaml:1051]: Bub?

**Review by allysonjp715 — APPROVED** [review]

File: verification/curator-service/api/openapi/openapi.yaml (hunk 1 above)
Comment (attwad) [reply verification/curator-service/api/openapi/openapi.yaml:988]: done.

File: verification/curator-service/api/openapi/openapi.yaml (hunk 2 above)
Comment (attwad) [reply verification/curator-service/api/openapi/openapi.yaml:1051]: sory tip0: meant Bus

**Pull request merged**